### PR TITLE
feat: Add prefer_name into ApisixClusterConfig

### PR DESCRIPTION
**Pull request by fatpa**
### Type of change:

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

Add `perfer_name` to apisixclusterconfig `prometheus.properties`

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

Refer to #1516

### CODE REVIEW

**Comment by fatpa** [comment]
Please take a look. @tao12345666333

**Comment by lingsamuel** [comment]
New features must include corresponding e2e tests.

**Comment by tao12345666333** [comment]
we have entered the rv1.6 elease window, I will put this to v1.7, thanks

**Comment by fatpa** [comment]
> we have entered the rv1.6 elease window, I will put this to v1.7, thanks

Ok, I will fix the e2e test bug first.

**Comment by fatpa** [comment]
New feature's e2e tests have run successfully. 

But there are some errors on the external-service e2e tests, it seems not linked to this PR.

**Comment by tao12345666333** [comment]
sorry for delay, re-run all test cases.

**Review by tao12345666333 — CHANGES_REQUESTED** [review]
Thanks,
we don't need change v2beta3.

**Comment by fatpa** [comment]
Need to rerun unit-test-ci.

File: utils/kind-with-registry.sh [hunk 1]
```diff
@@ -67,7 +67,7 @@ nodes:
 containerdConfigPatches:
 - |-
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
-    endpoint = ["http://${reg_host}:5000"]
+    endpoint = ["http://${reg_host}:${reg_port}"]
```
Comment (tao12345666333) [inline utils/kind-with-registry.sh:70]: I don't think adjustments are needed here.
Have you tested it? They use addresses inside the container

Comment (fatpa) [reply utils/kind-with-registry.sh]: My misunderstanding. Had reverted the script.

**Review by tao12345666333 — APPROVED** [review]

**Review by lingsamuel — APPROVED** [review]

**Pull request merged**